### ISSUE TITLE
Lack of background color contrast

### ISSUE THREAD

**Issue by jenniemc** [department-of-veterans-affairs/va.gov-team#24888]
## Summary
As a visually impaired user, i like to get a better color background indicator for the alert message about turning on browser locator in order to display community care provider  based on my current location. Once I tab away from the the alert message, it is no longer the active element, and thus loses the yellow outline making the light blue background indistinguishable to the grey background. 

## Specs
- Device: MAC
- Browser:  Chrome
- Finds from #24020 

## Steps to Reproduce
**Test User:** 

1. System preferences>Privacy & Security>Location Services> uncheck Google Chrome
2. Go to: /health-care/schedule-view-va-appointments/appointments/new-appointment/community-care-preferences
3. Click "Chose a provider"
4. Click "use your current location"

### Actual Result
The alert message initially has the yellow outline but once i tab to the next item, the alert message loses the outline making the light blue background indistinguishable to the grey background.

![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/c4bc48a9-965c-422c-9604-3c556e8e3f6a)

 ![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/d7447a84-f372-4bf3-993b-d89e14fdc662)

### Desired Result
an outline remains or the alert background color have greater contrast to the grey background

## Definition of Done
- [ ] increase color contrast or outline
- [ ] Relevant documentation is updated

---
## How to configure this issue
- [ ] **Labeled with Team** (`vaos`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `design`, `product`)
- [ ] **Labeled with** `Bug`
- [ ] **Assigned to** Product for prioritization

**Comment by outerpress** [comment]
Here's what I'm proposing for this change at a high level (see below for visual and spec-level deets)

Change the alert to [va-alert---warning component](https://design.va.gov/components/alertboxes#warning-alert)

Reasons:
- None of the background-only alerts will resolve the color contrast issue
- Reviewing the alert usage guidance, "warning" is most appropriate here - "Used to warn a user, such as when there are negative consequences, and when something has gone wrong."

----

Modify the alert to use a different background color

Reason:
- The gray background of the alert blends with the gray background of the card, making the contrast issue worse

----

Remove blue vertical bar and tweak content in address

Reason
- The blue bar doesn't add useful info or hierarchy and distracts from the important content on the page

----

Change details:

![24888 - Overview.png](https://images.zenhubusercontent.com/5f7ce3163782a585b910eaf3/47b28ab9-3223-4cbe-a023-3f8995e09db4)

**Comment by laurenernest** [comment]
Moving to dev to resolve per design changes [above](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24888#issuecomment-853188185)

**Comment by jenniemc** [comment]
@outerpress @laurenernest, There are other scenarios where the address has vertical blue bar along with the alert box message displaying browser blocking search for current location. How would you like to treat the below scenarios?

**New Appt Flow**
![image](https://user-images.githubusercontent.com/54327023/121189815-76b18000-c838-11eb-8890-676726388cce.png)

<hr />

**Covid Vaccination Flow**
![image](https://user-images.githubusercontent.com/54327023/121190150-c3955680-c838-11eb-9cc4-a9bcfae1f207.png)

**Comment by outerpress** [comment]
I'd love to remove the vertical blue bar from the address on this page entirely, no matter what the state (alert box or not). It doesn't seem to add anything useful, just noise on the page.

That said, @laurenernest is it worth doing this since we're looking at making some changes to this for our facilities page sorter experiment? I'm okay skipping this change if not.

**Comment by laurenernest** [comment]
If the facilities page won't inherit the CC change automatically, then we should wait for the sorting change.

@jenniemc is this in a component that changes everyone in the tool or does it require individual changes on each page?

**Comment by jenniemc** [comment]
@laurenernest, the changes to CC Provider selection pages will not be inherited by the other components impacting New VA Appointment Flow and  COVID Vaccine Flow. However, they are easy changes. Peter and Jeff agreed to proceed with the design updates to maintain consistency on the New VA Appointment choose location page and  COVID Vaccine choose location page.

Here is a preview of the COVID Vaccine choose location. 

cc @outerpress, @jbalboni 


![image](https://user-images.githubusercontent.com/54327023/121235220-9d84ac00-c862-11eb-8ca9-3bd4271b3a56.png)
